### PR TITLE
New package: notes 2.1.0

### PR DESCRIPTION
**Pull request by AgarimOSLinux**
- I tested the changes in this PR: YES

- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): YES

- I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

File: srcpkgs/notes/template [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Template file for 'notes'
+pkgname=notes
+version=2.1.0
+revision=1
+build_style=cmake
+configure_args="-DUSE_QT_VERSION=5 -DUPDATE_CHECKER=OFF"
+hostmakedepends="gcc git"
```
Comment (mhmdanas) [inline srcpkgs/notes/template:7]: gcc should be already installed in the masterdir:
```suggestion
hostmakedepends="git"
```

File: srcpkgs/notes/template [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+short_desc="Fast and beautiful note-taking app written in C++"
+maintainer="José Santos <agarimos@tutanota.com>"
+license="MPL-2.0"
+homepage="https://github.com/nuttyartist/notes"
+changelog="https://github.com/nuttyartist/${pkgname}/releases/tag/v${version}"
+distfiles="https://github.com/nuttyartist/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=8a8ae079fad6d9d5e9206acf5f4d19e1e73e1196e6c27408072b5cccf8778b95
+
+post_extract() {
+git clone https://github.com/alex-spataru/QSimpleUpdater.git --depth=1
```
Comment (mhmdanas) [inline srcpkgs/notes/template:18]: Is there no other way other than cloning these repos, especially manually in the template? Additionally,  the update checker is disabled in `configure_args`, so why is this repo in particular required?

Also, please indent your template properly (with tabs, not spaces!).

File: srcpkgs/notes/template [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+build_style=cmake
+configure_args="-DUSE_QT_VERSION=5 -DUPDATE_CHECKER=OFF"
+hostmakedepends="gcc git"
+makedepends="qt5-devel libSM-devel qt5-plugin-mysql qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-tds"
+short_desc="Fast and beautiful note-taking app written in C++"
+maintainer="José Santos <agarimos@tutanota.com>"
+license="MPL-2.0"
+homepage="https://github.com/nuttyartist/notes"
+changelog="https://github.com/nuttyartist/${pkgname}/releases/tag/v${version}"
+distfiles="https://github.com/nuttyartist/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```
Comment (mhmdanas) [inline srcpkgs/notes/template:14]:
```suggestion
changelog="https://github.com/nuttyartist/notes/releases"
distfiles="https://github.com/nuttyartist/notes/archive/refs/tags/v${version}.tar.gz"
```

**Review by mhmdanas — CHANGES_REQUESTED** [review]

File: srcpkgs/notes/template (hunk 2 above)
Comment (AgarimOSLinux) [reply srcpkgs/notes/template:18]: https://github.com/nuttyartist/notes/tree/master/3rdParty
Without them the build will fail.

Comment (classabbyamp) [reply srcpkgs/notes/template:18]: > Is there no other way other than cloning these repos, especially manually in the template?

the proper way is to include them as distfiles

**Pull request closed without merging**